src/page/MyNotes.jsx

```diff
@@ -259,7 +261,15 @@ function MyNotes() {
 
       <NoteContainer>
         {notes.map((note) => (
-          <Note key={note.id} note={note} />
+
+          <Note
```

Comment:
Det her virker ikke. I den gamle version repræsenterer note hele noteDTO'en sent fra backend, dvs. den indeholder allerede alle atributer og du behøver ikke sende dem ind hver for sig. Udover det sætter du note til at være note.content.
Den skal se sådan ud(ligesom før:
<Note key={note.id} note={note} />